src/data.js

```diff
@@ -2265,6 +2265,26 @@ module.exports = [
       'UX',
       'Open Source',
     ],
+  },
+      {
+    name: 'Sean Smith',
+    description: 'Developer, Front End Developer, Craft CMS enthusiast, and podcaster',
+    url: 'https://caffeinecreations.ca/uses',
+    twitter: '@cs_sean',
+    emoji: '',
+    country: 'CA',
+    computer: 'windows',
+    phone: 'android',
+    tags: [
+      'Developer',
+      'Front End',
+      'Accessibility',
+      'CSS',
+      'Craft CMS',
+      'JavaScript',
+      'podcaster',
+      'podcast'
```

Comment:
Please remove one of these tags - I think "podcaster" is the more common one in this project